### ISSUE TITLE
The Appender class was deprecated in Matplotlib 3.1 and will be removed in 3.3. Warning

### ISSUE THREAD

**Issue by mariopareto** [python-windrose/windrose#119]
After updating the matplotlib to 3.1, the windrose package rises a warning about the appender class:

C:\Users\mariod\AppData\Local\Continuum\anaconda3\envs\defaultenv\lib\site-packages\windrose\windrose.py:29: MatplotlibDeprecationWarning: 
The Appender class was deprecated in Matplotlib 3.1 and will be removed in 3.3.
  addendum = docstring.Appender(msg, "\n\n")

**Comment by scls19fr** [comment]
Thanks for reporting.
Could you ask to Matplotlib devs what should be used instead?

**Comment by dancergraham** [comment]
This was fixed in https://github.com/python-windrose/windrose/pull/118 and can now be closed, as far as I can see (tested on Python 3.7 for Windows)

**Comment by scls19fr** [comment]
oh yes @dancergraham ! it looks duplicate of #117 which was already fixed (by PR #118)

**Comment by AndyLuckert** [comment]
I'm sorry to tell you that **the issue came back**, but this time not as a warning: **now it's become an error** which prevents the scripts from being executed properly:

```
~/Dokumente/Allgemeines_material/Sonstiges/Programming/Python/Scripts/General/Plotting/plotting.py in <module>
     32 import pdfkit
     33 import imgkit
---> 34 from windrose import WindroseAxes
     35 from windrose import plot_windrose
     36 from tqdm import tqdm

/home/linuxbrew/.linuxbrew/lib/python3.8/site-packages/windrose/__init__.py in <module>
      4 from __future__ import absolute_import, division, print_function  # noqa
      5 
----> 6 from .windrose import WindAxesFactory  # noqa
      7 from .windrose import WindroseAxes  # noqa
      8 from .windrose import WindAxes  # noqa

/home/linuxbrew/.linuxbrew/lib/python3.8/site-packages/windrose/windrose.py in <module>
    760 
    761 
--> 762 @_autogen_docstring(WindroseAxes.contour)
    763 def wrcontour(direction, var, ax=None, rmax=None, **kwargs):
    764     """

/home/linuxbrew/.linuxbrew/lib/python3.8/site-packages/windrose/windrose.py in _autogen_docstring(base)
     27     # msg = "\n\nAdditional kwargs: hold = [True|False] overrides default hold state"
     28     msg = ""
---> 29     addendum = docstring.Appender(msg, "\n\n")
     30     return lambda func: addendum(docstring.copy_dedent(base)(func))
     31 

AttributeError: module 'matplotlib.docstring' has no attribute 'Appender'
```

I assume that this changed after having updated all python packages like 1 hour ago.
Now, the versions of windrose and matplotlib are the following:

```
andylu@andylu-ThinkPad-Edge-E130:~/Desktop/Python/Scripts$ pip show windrose
Name: windrose
Version: 1.6.7
Summary: Python Matplotlib, Numpy library to manage wind data, draw windrose (also known as a polar rose plot)
Home-page: https://github.com/python-windrose/windrose
Author: Lionel Roubeyrie
Author-email: s.celles@gmail.com
License: CeCILL-B OR BSD-3-Clause
Location: /home/linuxbrew/.linuxbrew/lib/python3.8/site-packages
Requires: matplotlib, numpy
Required-by: 

andylu@andylu-ThinkPad-Edge-E130:~/Desktop/Python/Scripts$ pip show matplotlib
Name: matplotlib
Version: 3.3.0
Summary: Python plotting package
Home-page: https://matplotlib.org
Author: John D. Hunter, Michael Droettboom
Author-email: matplotlib-users@python.org
License: PSF
Location: /home/linuxbrew/.linuxbrew/lib/python3.8/site-packages
Requires: pillow, python-dateutil, kiwisolver, numpy, cycler, pyparsing
Required-by: windrose, seaborn, osmnx, descartes
```

How can I fix this issue?

**Comment by dancergraham** [comment]
https://github.com/python-windrose/windrose/issues/136#issuecomment-662042174

**Comment by AndreasLuckert** [comment]
It's true that the mentioned workaround `pip install git+https://github.com/python-windrose/windrose` makes the error vanish, but nonetheless I'd be great if the standard installation command `pip install windrose` made it work directly.

**Comment by weber-s** [comment]
Hello,

I should be fixed with the latest pypi version:

```
pip3 install windrose --upgrade
```
